projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -0,0 +1,114 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
+import { SubscriptionLifecycle, TypedSimpleChanges } from '@hypertrace/common';
+import { Subject } from 'rxjs';
+import { debounceTime } from 'rxjs/operators';
+import { ToggleItem } from '../../toggle-group/toggle-item';
+import { TableMode } from '../table-api';
+
+@Component({
+  selector: 'ht-table-controls',
+  styleUrls: ['./table-controls.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="table-controls">
+      <!-- Filters -->
+      <div class="filter-controls">
+        <!-- Search -->
+        <ht-search-box
+          *ngIf="this.searchAttribute"
+          class="control search-box"
+          [placeholder]="this.searchPlaceholder"
+          (valueChange)="this.onSearchChange($event)"
+        ></ht-search-box>
+
+        <!-- Filter Toggle -->
+        <ht-toggle-group
+          *ngIf="this.filterItems?.length"
+          class="control filter-toggle-group"
+          [items]="this.filterItems"
+          [activeItem]="this.activeFilterItem"
+          (activeItemChange)="this.onFilterChange($event)"
+        ></ht-toggle-group>
+      </div>
+
+      <!-- Mode Toggle -->
+      <ht-toggle-group
+        *ngIf="this.modeItems?.length"
+        class="control mode-toggle-group"
+        [items]="this.modeItems"
+        [activeItem]="this.activeModeItem"
+        (activeItemChange)="this.onModeChange($event)"
+      ></ht-toggle-group>
+    </div>
+  `
+})
+export class TableControlsComponent implements OnChanges {
+  @Input()
+  public searchAttribute?: string;
```

Comment:
This should either be used to emit a search filter, or be removed in favor of a boolean, since that's the only way it's being used currently

Reply:
Fixed